File: tests/fixtures/boundary_ir/repos/python/app/service.py
from app.formatting import format_name


class Renderer:
    def render(self, value: str) -> str:
        return format_name(value)


def render_report(value: str) -> str:
    from app.formatting import format_name

    cleaned = format_name(value)
    duplicate = helper(cleaned)  # noqa: F821 - unresolved-call fixture
    return missing_call(duplicate)  # noqa: F821 - unresolved-call fixture

File: tests/fixtures/boundary_ir/repos/typescript/service.ts
import { formatName } from './format';
import type { Formatter } from './format';

export class Renderer implements Formatter {
  render(value: string): string {
    return formatName(value);
  }
}

export function run(value: string): string {
  const cleaned = formatName(value);
  const duplicate = helper(cleaned);
  return missingCall(duplicate);
}
